### PR TITLE
feat: add storefront to app server communication

### PR DESCRIPTION
**Pull request by shyim**

### CODE REVIEW

**Comment by JoshuaBehrens** [comment]
This will add so much to cookie and data security topics :/

File: guides/plugins/apps/clientside-to-app-backend.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+```mermaid
+sequenceDiagram
+    participant Client
+    participant Shopware Backend
+    participant App Server
+    Client->>Shopware Backend: POST /store-api/app-system/MyApp/generate-token
+    Shopware Backend->>Client: Responds with Signed JWT Token
+    Client->>App Server: Post /product-review/submit containing JWT in header
+```
```
Comment (sushmangupta) [inline guides/plugins/apps/clientside-to-app-backend.md:19]: @shyim , Currently, Gibook doesn't support mermaid conversions. Can you please provide a jpeg image as an alternative ?

File: guides/plugins/apps/clientside-to-app-backend.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
+# Client side communication to the app backend
+
+To give some context to the request from the browser directly to the app backend, we allow generating a JWT token that contains session information as claims and is signed by the shop.
```
Comment (sushmangupta) [inline guides/plugins/apps/clientside-to-app-backend.md:3]:
```suggestion
Direct communication from the browser to the app backend involves generating a JWT token. This token contains session-specific information, as [claims](#the-jwt-token), and is securely signed by the shop. This mechanism ensures a secure exchange of data between the client and the app backend.
```

**Review by sushmangupta — CHANGES_REQUESTED** [review]

File: guides/plugins/apps/clientside-to-app-backend.md (hunk 1 above)
Comment (sushmangupta) [reply guides/plugins/apps/clientside-to-app-backend.md:19]: Ignore the above comment. Gitbook now supports mermaid plugin 👍

**Comment by shyim** [comment]
@JoshuaBehrens 

it has nothing to do with cookies (we don't have some, everything stateless), but data security topics you will anyhow have.

**Review by sushmangupta — APPROVED** [review]

**Pull request merged**